app/controllers/my_facilities/ranked_facilities_controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
```

Comment:
Should we add some controller specs for this as well?

Reply:
We're iterating on this so I skipped controller/integration specs for now.